2025-03-05 03:10:09,066 INFO: Starting application in development mode [in F:\PythonII\Dating\app\utils\logging.py:43]
2025-03-05 03:10:10,010 WARNING:  * Debugger is active! [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:10:10,014 INFO:  * Debugger PIN: 132-194-637 [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:10:12,676 INFO: Successful login for user: 0000000 [in F:\PythonII\Dating\app\services\auth_service.py:34]
2025-03-05 03:10:12,679 INFO: 127.0.0.1 - - [05/Mar/2025 03:10:12] "[32mPOST /login HTTP/1.1[0m" 302 - [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:10:12,699 INFO: 127.0.0.1 - - [05/Mar/2025 03:10:12] "[32mGET / HTTP/1.1[0m" 302 - [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:10:12,771 INFO: 127.0.0.1 - - [05/Mar/2025 03:10:12] "[35m[1mGET /profile HTTP/1.1[0m" 500 - [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:10:12,936 INFO: 127.0.0.1 - - [05/Mar/2025 03:10:12] "[36mGET /profile?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 - [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:10:12,937 INFO: 127.0.0.1 - - [05/Mar/2025 03:10:12] "[36mGET /profile?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 - [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:10:13,098 INFO: 127.0.0.1 - - [05/Mar/2025 03:10:13] "GET /profile?__debugger__=yes&cmd=resource&f=console.png&s=41Ayc3Z3H8026gILTUnH HTTP/1.1" 200 - [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:10:13,104 INFO: 127.0.0.1 - - [05/Mar/2025 03:10:13] "[36mGET /profile?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:10:13,324 INFO: 127.0.0.1 - - [05/Mar/2025 03:10:13] "[36mGET /profile?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
2025-03-05 03:11:11,884 INFO:  * Detected change in 'F:\\PythonII\\Dating\\app\\admin\\__init__.py', reloading [in F:\PythonII\Dating\.venv\Lib\site-packages\werkzeug\_internal.py:97]
